sonar-plugin-api/src/main/java/org/sonar/api/resources/DuplicatedSourceException.java

```diff
@@ -24,7 +24,9 @@
 
 /**
  * @since 2.6
+ * @deprecated since 5.6 should not be used in any API
  */
+@Deprecated
 public final class DuplicatedSourceException extends SonarException {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)